2019-12-18 18:37:39,405 - root - INFO - you had select Chrome browser.
2019-12-18 18:37:39,406 - root - INFO - The test URL is: http://bing.com
2019-12-18 18:37:42,214 - root - INFO - Open url: http://bing.com
2019-12-18 18:37:42,216 - root - INFO - Set implicitly wait 10 seconds.
2019-12-18 18:37:42,287 - BasePage - INFO - Had find the element '  ' successful by xpath via value: //*[@id="sb_form_q"] 
2019-12-18 18:37:42,287 - BasePage - INFO - Had find the element '  ' successful by xpath via value: //*[@id="sb_form_q"] 
2019-12-18 18:37:42,550 - BasePage - INFO - Had type ' selenium ' in inputBox
2019-12-18 18:37:42,550 - BasePage - INFO - Had type ' selenium ' in inputBox
2019-12-18 18:37:42,724 - BasePage - INFO - Had find the element '  ' successful by xpath via value: //*[@id="sb_form_q"] 
2019-12-18 18:37:42,724 - BasePage - INFO - Had find the element '  ' successful by xpath via value: //*[@id="sb_form_q"] 
2019-12-18 18:37:42,733 - BasePage - INFO - Clear text in input box before typing.
2019-12-18 18:37:42,733 - BasePage - INFO - Clear text in input box before typing.
2019-12-18 18:37:42,780 - BasePage - INFO - Had find the element ' 国际版 ' successful by id via value: est_en 
2019-12-18 18:37:42,780 - BasePage - INFO - Had find the element ' 国际版 ' successful by id via value: est_en 
2019-12-18 18:37:42,793 - BasePage - INFO - The element ' 国际版 ' was clicked.
2019-12-18 18:37:42,793 - BasePage - INFO - The element ' 国际版 ' was clicked.
2019-12-18 18:37:44,223 - BasePage - INFO - Had find the element '  ' successful by xpath via value: //*[@id="sb_form_q"] 
2019-12-18 18:37:44,223 - BasePage - INFO - Had find the element '  ' successful by xpath via value: //*[@id="sb_form_q"] 
2019-12-18 18:37:44,347 - BasePage - INFO - Had type ' python ' in inputBox
2019-12-18 18:37:44,347 - BasePage - INFO - Had type ' python ' in inputBox
2019-12-18 18:37:44,384 - BasePage - INFO - Had find the element '  ' successful by xpath via value: /html/body/table/tbody/tr/td/div/div[2]/div[3]/form/div/div[1]/input 
2019-12-18 18:37:44,384 - BasePage - INFO - Had find the element '  ' successful by xpath via value: /html/body/table/tbody/tr/td/div/div[2]/div[3]/form/div/div[1]/input 
2019-12-18 18:37:44,399 - BasePage - INFO - The element '  ' was clicked.
2019-12-18 18:37:44,399 - BasePage - INFO - The element '  ' was clicked.
2019-12-18 18:37:49,161 - urllib3.connectionpool - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'ProtocolError('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))': /session/a6205e9dc020904154cee93f245b795c/screenshot
2019-12-18 18:37:49,164 - urllib3.connectionpool - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x105dbe810>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/a6205e9dc020904154cee93f245b795c/screenshot
2019-12-18 18:37:49,165 - urllib3.connectionpool - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x105dbe9d0>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/a6205e9dc020904154cee93f245b795c/screenshot
2019-12-18 18:37:49,167 - urllib3.connectionpool - WARNING - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x105dbeb10>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/a6205e9dc020904154cee93f245b795c
2019-12-18 18:37:49,169 - urllib3.connectionpool - WARNING - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x105f5a510>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/a6205e9dc020904154cee93f245b795c
2019-12-18 18:37:49,171 - urllib3.connectionpool - WARNING - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x105f5a5d0>: Failed to establish a new connection: [Errno 61] Connection refused')': /session/a6205e9dc020904154cee93f245b795c
